docker/scripts/start-kuzzle-dev.ts

```diff
@@ -38,7 +39,7 @@ async function loadAdditionalPlugins() {
   }
 }
 
-if (! process.env.TRAVIS) {
+if (!process.env.TRAVIS) {
```

Comment:
I think you can remove this condition or use the `CI` environment variable:
```suggestion
if (!process.env.CI) {
```